### ISSUE TITLE
Carton/CLI.pm

### ISSUE THREAD

**Issue by valderaplets** [YunoHost-Apps/lstu_ynh#47]
Hello, have you encountered this error before?

```
Aug 14 19:01:34 yunohost carton[3094]: Can't locate Carton/CLI.pm in @INC (you may need to install the Carton::CLI module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.28.1 /usr/local/share/perl/5.28.1 /usr/lib/x86_64-linux-gnu/perl5/5.28 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.28 /usr/share/perl/5.28 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/local/bin/carton line 4.
Aug 14 19:01:34 yunohost carton[3094]: BEGIN failed--compilation aborted at /usr/local/bin/carton line 4.
```

same issue with lufi

**Comment by ldidry** [comment]
I’m sure that you ugraded to Buster, didn’t you? Run `cpanm Carton` (or `cpan Carton` if `cpanm` is not available) to reinstall Carton.

**Comment by valderaplets** [comment]
Okay, I run a cpan carton, and now : 
`Aug 15 09:44:24 yunohost carton[3566]: ListUtil.c: loadable library and perl binaries are mismatched (got handshake key 0xdb80080, needed 0xce00080)`
:(

**Comment by ldidry** [comment]
See #46 
TL; DR:
```
rm -rf local
carton install --deployment --without=sqlite --without=mysql
```
(do that in Lstu installation directory)

**Comment by valderaplets** [comment]
Thank you it worked for both lstu and lufi.
thanks for your help
